pilot/pkg/networking/core/v1alpha3/cluster_test.go

```diff
@@ -905,17 +904,7 @@ func TestBuildAutoMtlsSettings(t *testing.T) {
 			[]string{"spiffe://foo/serviceaccount/1"},
 			"foo.com",
 			&model.Proxy{Metadata: &model.NodeMetadata{}},
-			false, false, model.MTLSDisable, cluster.Cluster_EDS,
-			nil,
```

Comment:
add a new test case for new behavior at least?

Reply:
The existing cases have covered the MTLSDisable scene, this infer is tested by integration test

Reply:
sure. won't do any harm to add unit test coverage as well, right?

Reply:
done